package.json

```diff
@@ -112,6 +112,7 @@
     "gifshot": "^0.4.5",
     "lodash": "^4.17.14",
     "mathjs": "^6.6.4",
-    "moment": "^2.24.0"
+    "moment": "^2.24.0",
+    "wavesurfer.js": "3.3.3"
```

Comment:
Can you please create an issue to add '^` here once we have the warning functionality? [This will help us to track this task]

Reply:
Done #9287